tests/container_checker/test_container_checker.py

```diff
@@ -99,7 +99,7 @@ def get_disabled_container_list(duthost):
     pytest_assert(succeeded, "Failed to get status ('enabled'|'disabled') of containers. Exiting...")
 
     for container_name, status in container_status.items():
-        if status == "disabled":
+        if "disabled" in status:
```

Comment:
Last week, I installed 202012 image and found the `state` field of `mux` container is set to `always_disabled`, do we need also check `always_disabled` at here?